integration/js/utils/SdkBaseTest.js

```diff
@@ -20,6 +20,10 @@ class SdkBaseTest extends KiteBaseTest {
       this.userArn = this.payload.userArn;
     }
 
+    if (['Video', 'Audio'].includes(testName)) {
```

Comment:
Just double checking - this is not backwards? Will 'testname' be Video or Audio?

Reply:
Just checked - our Audio and Video E2E test use the name "Audio" and "Video". Thanks